### ISSUE TITLE
[Bug]: Cannot install in python 3.12 with pyside6

### ISSUE THREAD

**Issue by cmbs2019** [zhiyiYo/PyQt-Fluent-Widgets#784]
### What happened?

pyside6-fluent-widgets[full] 1.5.1 depends on PySide6<=6.4.2
but pyside6 >6.5.0 in python3.12
how to solve?

### Operation System

Windows 11 23H2

### Python Version

3.12

### PyQt/PySide Version

pyside6 6.6.2

### PyQt/PySide-Fluent-Widgets Version

6.5.1

### How to Reproduce?

Install pyside-fluent-widgets in python3.12

### Minimum code

```python3
Using cached PySide6_Fluent_Widgets-0.8.9-py3-none-any.whl.metadata (4.5 kB)
ERROR: Cannot install pyside6-fluent-widgets==0.10.0, pyside6-fluent-widgets==0.10.1, pyside6-fluent-widgets==0.8.9, pyside6-fluent-widgets==0.9.0, pyside6-fluent-widgets==0
.9.1, pyside6-fluent-widgets==0.9.10, pyside6-fluent-widgets==0.9.2, pyside6-fluent-widgets==0.9.3, pyside6-fluent-widgets==0.9.4, pyside6-fluent-widgets==0.9.5, pyside6-flu
ent-widgets==0.9.6, pyside6-fluent-widgets==0.9.7, pyside6-fluent-widgets==0.9.8, pyside6-fluent-widgets==0.9.9, pyside6-fluent-widgets==1.0.0, pyside6-fluent-widgets==1.0.1
, pyside6-fluent-widgets==1.0.2, pyside6-fluent-widgets==1.0.3, pyside6-fluent-widgets==1.0.4, pyside6-fluent-widgets==1.1.1, pyside6-fluent-widgets==1.1.2, pyside6-fluent-w
idgets==1.1.3, pyside6-fluent-widgets==1.1.4, pyside6-fluent-widgets==1.1.6, pyside6-fluent-widgets==1.1.7, pyside6-fluent-widgets==1.1.8, pyside6-fluent-widgets==1.1.9, pys
ide6-fluent-widgets==1.2.0, pyside6-fluent-widgets==1.2.1, pyside6-fluent-widgets==1.2.2, pyside6-fluent-widgets==1.2.3, pyside6-fluent-widgets==1.2.4, pyside6-fluent-widget
s==1.2.6, pyside6-fluent-widgets==1.2.7, pyside6-fluent-widgets==1.2.8, pyside6-fluent-widgets==1.2.9, pyside6-fluent-widgets==1.3.0, pyside6-fluent-widgets==1.3.1, pyside6-
fluent-widgets==1.3.2, pyside6-fluent-widgets==1.3.3, pyside6-fluent-widgets==1.3.4, pyside6-fluent-widgets==1.3.5, pyside6-fluent-widgets==1.3.6, pyside6-fluent-widgets==1.
3.7, pyside6-fluent-widgets==1.3.8, pyside6-fluent-widgets==1.4.0, pyside6-fluent-widgets==1.4.1, pyside6-fluent-widgets==1.4.2, pyside6-fluent-widgets==1.4.3, pyside6-fluen
t-widgets==1.4.5, pyside6-fluent-widgets==1.4.6, pyside6-fluent-widgets==1.4.7, pyside6-fluent-widgets==1.5.0 and pyside6-fluent-widgets==1.5.1 because these package versions have conflicting dependencies.
```

**Comment by AlexZhu2001** [comment]
Due to #461 , the pyside version is limited to be less than 6.4.2. This bug has been fixed in 6.6.2 and the requirements will be updated in the next version
You can clone and modify it for yourself temporarily.

**Comment by zhiyiYo** [comment]
1.5.2 已取消 pyside6 版本限制
